tests/integration_tests/validation/conformance_suite_configurations/kvk_nt16.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+from pathlib import PurePath
+from tests.integration_tests.validation.conformance_suite_config import ConformanceSuiteConfig
+
+ZIP_PATH = 'NT16_KVK_20211208 Berichten_0.zip'
+# needs to be extracted because arelle can't load a taxonomy package ZIP from within a ZIP
+EXTRACTED_PATH = ZIP_PATH.replace('.zip', '')
+config = ConformanceSuiteConfig(
+    args=[
+        '--disclosureSystem', 'NT16-preview',
+    ],
+    cache_version_id='VMevF85TT4vp4NPqHeTOB2RjFGCU8bQY',
```

Comment:
This version ID should match a version of the `kvk_nt16.zip` cache package in arelle-public. This doesn't use the private bucket so you can test locally by using `--download-cache` with the conformance suite runner.